src/SubscriptionServer.ts

```diff
@@ -40,7 +43,7 @@ export default class SubscriptionServer<TContext, TCredentials> {
       config.createLogger || defaultCreateLogger;
     this.log = createLogger('@4c/SubscriptionServer::Server');
 
-    this.io = new IoServer({
+    this.io = IoServer({
```

Comment:
Instantiating a socketio server vs using the default static one? types aren't providing the `new` constructor